aws-aurora-postgresql.yml

```diff
@@ -105,12 +103,25 @@ provision:
     constraints:
       minimum: 1
       maximum: 35
-  - <<: *nullable_string
-    field_name: preferred_backup_window
+  - field_name: preferred_backup_window
```

Comment:
**question**: Is this revert intentional?

Reply:
That's my fault!! I just changed it. Since engine_version is no longer null, I will define the nullable_string structure just associated with the preferred_backup_window